examples/albert/model.py

```diff
@@ -27,6 +28,7 @@ def schedule_model(
     group=None,
     bcast_input=False,
     pipeline_cuts=None,
+    separate_op_fusion=False,
```

Comment:
Maybe directly using `disable_fuse_bias_gelu` would be more clear and align to `disable_flash_attn`.